src/NHPersistenceCompatibilityTests/Common/NHibernateConnectionInfo.cs

```diff
@@ -4,8 +4,8 @@ namespace Common
 {
     public class NHibernateConnectionInfo
     {
-        const string Dialect = "NHibernate.Dialect.MsSql2012Dialect";
-        const string ConnectionString = @"Data Source=localhost\SQLEXPRESS;Initial Catalog=persistencetests;Integrated Security=True";
+        public const string Dialect = "NHibernate.Dialect.MsSql2012Dialect";
+        public const string ConnectionString = @"Data Source=localhost\SQLEXPRESS;Initial Catalog=persistencetests;Integrated Security=True";
```

Comment:
That'd be a bit tricky. Putting this into app.configs we have to put them into app.config of each project/version, which means more duplicates. Furthermore, this is executed using AppDomains which means more trouble loading the right app.config for a given AppDomain. The simplest approach would be just having one shared connection-string in code. Should we need to configure it, I'd go with environment variables instead of app.config approach.